contracts/swap/Cargo.toml

```diff
@@ -1,25 +1,25 @@
 [package]
-name = "helix-converter"
-version = "0.1.0"
-authors = ["Antoni Mysliborski <antoni@injectivelabs.org>"]
+authors = [ "Antoni Mysliborski <antoni@injectivelabs.org>" ]
 edition = "2021"
+name    = "helix-converter"
```

Comment:
Can we change it to injective-converter , this way we make it generic for all products that might use, being Helix and Mito initially.